uvtools/plot.py

```diff
@@ -723,3 +723,185 @@ def plot_diff_uv(uvd1, uvd2, pol=None, check_metadata=True, bins=50):
         fig.colorbar(cax)
 
     return fig
+
+def plot_diff_1d(uvd1, uvd2, antpairpol, plot_type="both", 
+                 check_metadata=True, dimension=None):
+    # TODO: docstring
+    """Produce plots of visibility differences along a single axis.
+
+    Parameters
+    ----------
+    uvd1, uvd2 : pyuvdata.UVData
+        UVData objects which store visibilities to be differenced and their
+        associated metadata. They should have the same number of frequencies,
+        same baselines, and same times as each other.
+
+    antpairpol : tuple
+        Tuple specifying which baseline and polarization to use to compare 
+        visibility waterfalls. See pyuvdata.UVData.get_data method docstring 
+        for information on accepted tuples.
+    
+    plot_type : str, optional
+        A string identifying which quantities to plot. Accepted values are 
+        as follows:
+            - base
+                - Single row of plots in the usual basis (time or frequency).
+            - dual
+                - Single row of plots in Fourier space (fringe rate or delay).
+            - both
+                - Two rows of plots in the usual and Fourier domains.
+        Default behavior is to use the 'both' setting.
+
+    check_metadata : bool, optional
+        Whether to check that the metadata for `uvd1` and `uvd2` match.
+        See ``utils.check_uvd_pair_metadata`` docstring for details on 
+        how the metadata are compared. If `check_metadata` is set to 
+        False, but the metadata don't agree, then the plotter may or 
+        may not error out, depending on how the metadata disagree. 
+        Default behavior is to check the metadata.
+    
+    dimension : str, optional
+        String specifying which dimension is used for the base domain. This 
```

Comment:
"base domain" -- could be a bit clearer what this means

Reply:
Does the note that this should be either `'time'` or `'freq'` not clarify enough? Do you have a suggestion for what I could add to make it clearer?